pkg/controller/stacks/stack/stack_test.go

```diff
@@ -1339,6 +1356,107 @@ func TestProcessDeployment(t *testing.T) {
 				},
 			},
 		},
+		{
+			name: "SuccessHostedTruncated",
+			r: resource(
+				withNamespacedName(types.NamespacedName{
+					Name:      resourceName + dashAlphabet,
+					Namespace: namespace + dashAlphabet,
+				}),
+				withControllerSpec(defaultControllerSpec())),
+			initObjs: []runtime.Object{sa(
+				withSANamespacedName(types.NamespacedName{
+					Name:      resourceName + dashAlphabet,
+					Namespace: namespace + dashAlphabet,
+				}),
+				withTokenSecret(corev1.ObjectReference{
+					Name:      resourceName + dashAlphabet,
+					Namespace: namespace + dashAlphabet,
+				})), saSecret(resourceName+dashAlphabet, namespace+dashAlphabet)},
```

Comment:
Nitpick: noticing the pattern again of having no spaces around the operator

Reply:
VSCode is collapsing these spaces. I'm going to let this stand since I can not easily split these into separate declarations in the test cases.  If I make each combination into a new constant the names would be even longer: `resourceNamePlusDashAlphabet` :stuck_out_tongue:

Reply:
`go fmt` seems to be doing this when there are multiple arguments to a function:
https://play.golang.org/p/xVTwhIh6eAb

Reply:
VS Code collapsing the spaces implies to me that that is the idiomatic thing to do :)